### PR TITLE
Organise libero package

### PR DESCRIPTION
**Pull request by GiancarloFusiello**
This PR provides a little house keeping.

- Move `aws.py` and `task_helpers.py` to the `libero` package
- Rename `task_helpers.py` to `context_facades.py`

### CODE REVIEW

File: tests/test_task_helpers.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import pytest
```
Comment (giorgiosironi) [inline tests/test_task_helpers.py]: shouldn't this file be renamed to `test_context_facades.py` or `test_libero_context_facades.py`?

**Comment by giorgiosironi** [comment]
(Working on https://github.com/libero/jats-ingester/pull/56 for the build problem)

File: tests/test_task_helpers.py (hunk 1 above)
Comment (diversemix) [reply tests/test_task_helpers.py]: surely `libero` is redundant in any filename?

Comment (GiancarloFusiello) [reply tests/test_task_helpers.py]: Good catch. Renamed to `test_context_facades.py`.

**Pull request merged**